### PR TITLE
Fixing bug that prevents to add a new pin - close #1

### LINKED ISSUE #1: Fixing bug that prevents to add a new pin
Bug reproduction:
- follow the README to launche a local instance
- try to add a new pin on http://localhost

### PR DESCRIPTION
**Pull request by Lucas-C**
@Thomas-dot-G @xou816 : could you review this please :)

### CODE REVIEW

File: vboard-front/httpd.conf [hunk 1]
```diff
@@ -564,9 +564,9 @@ SSLRandomSeed connect builtin
 
   ## Proxy rules
   ProxyRequests Off
-  <Location /api/v1>
+  <Location ${$VBOARD_API_ENDPOINT}>
```
Comment (xou816) [inline vboard-front/httpd.conf]: i think an extra dollar got in there!

File: vboard-front/run_http_server.sh [hunk 2]
```diff
@@ -7,10 +7,24 @@ if ! [ -d /var/www/vboard/compile ]; then
     exit 1
 fi
 
-echo 'Inserting $VBOARD_WP_PUBLIC_HOST or /vblog in config.js'
-sed -i "s~\$VBOARD_WP_PUBLIC_HOST~${VBOARD_WP_PUBLIC_HOST:-/vblog}~" /var/www/vboard/compile/scripts/config.js
-echo 'Inserting $VBOARD_LOCALISATIONS in config.js'
-sed -i "s/\$VBOARD_WP_PUBLIC_HOST/${VBOARD_LOCALISATIONS:-}/" /var/www/vboard/compile/scripts/config.js
+if [ -n "${VBOARD_API_ENDPOINT:-}" ]; then
+    echo 'Inserting $VBOARD_API_ENDPOINT in index.html'
+    sed -i "s/\$VBOARD_API_ENDPOINT/${VBOARD_API_ENDPOINT:-}/" /var/www/vboard/index.html
```
Comment (xou816) [inline vboard-front/run_http_server.sh]: VBOARD_API_ENDPOINT includes forward slashes, sed calls should use tildes like below

**Comment by xou816** [comment]
Apart from the two small issues above, adding a pin seems to work fine! :)
Pin editing still has some issues however (I'll look into it and open a new issue if needed)

File: vboard-front/run_http_server.sh (hunk 2 above)
Comment (Lucas-C) [reply vboard-front/run_http_server.sh]: Well spotted !

**Comment by Lucas-C** [comment]
Yes, I saw an issue with image uploading too in pin creation.
I'm going to merge your PR to fix issue #24 an see if it improves things

**Pull request merged**